rfc-report.py

```diff
@@ -49,9 +48,7 @@ def github_comments(pull):
 	state = pull["state"]
 	labels = {label["name"] for label in pull["labels"]}
 	title = pull["title"]
-	if pull["draft"]:
-		in_draft.append(pull)
```

Comment:
I think just removing this will have them end up in `in_new`. We need to explicitly filter them out (ideally via the API query, but if not just adding `pass` here should work.

Reply:
There doesn't appear to be a draft filter in the query so just `pass`ing here is probably the best option.